azavea/ism-watershed-wellness-snapshot
# Transform sensor data to quality values

## Overview

Add functionality to transform the sensor data into water quality ratings which can be used by other parts of the app to convey the health of each waterbody.

A few tests for the water quality ratings are added for good measure.

The healthy ranges are placeholders for the moment as we await real data from the client.

Connects #60 

### Demo

![image](https://user-images.githubusercontent.com/1042475/53584187-17d90880-3b51-11e9-8ae6-e7d673333f8c.png)

![image](https://user-images.githubusercontent.com/1042475/53584150-0132b180-3b51-11e9-8947-4c1e515586a4.png)

## Testing Instructions

- Run the tests with `./scripts/tests`, and verify they all pass.
- Run the server, and visit the site. Look through the actions, and verify that the state `sensorRatings` key is updated according to the details laid out in #62.
